nipype/interfaces/io.py

```diff
@@ -1024,15 +1024,18 @@ def _list_outputs(self):
     def s3tolocal(self, s3path, bkt):
         import boto
         # path formatting
-        if not os.path.split(self.inputs.local_directory)[1] == '':
-            self.inputs.local_directory += '/'
-        if not os.path.split(self.inputs.bucket_path)[1] == '':
-            self.inputs.bucket_path += '/'
-        if self.inputs.template[0] == '/':
-            self.inputs.template = self.inputs.template[1:]
-
-        localpath = s3path.replace(self.inputs.bucket_path,
-                                   self.inputs.local_directory)
+        local_directory = str(self.inputs.local_directory)
+        bucket_path = str(self.inputs.bucket_path)
+        template = str(self.inputs.template)
+        if not os.path.split(local_directory)[1] == '':
```

Comment:
minor refactor for added clarity
```suggestion
        if not os.path.basename(local_directory) == '':
```